src/pages/index.tsx

```diff
@@ -1,40 +1,51 @@
-import React from 'react';
-import Link from '@docusaurus/Link';
-import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
-import Layout from '@theme/Layout';
+import React from "react";
+import Link from "@docusaurus/Link";
+import useDocusaurusContext from "@docusaurus/useDocusaurusContext";
+import Layout from "@theme/Layout";
 
-import styles from './index.module.css';
+import styles from "./index.module.css";
+
+import Translate, { translate } from "@docusaurus/Translate";
 
 function HomepageHeader() {
-  const {siteConfig} = useDocusaurusContext();
+  const { siteConfig } = useDocusaurusContext();
   return (
     <header className={styles.heroBanner}>
       <div className="container">
         <h1 className="hero__title">{siteConfig.title}</h1>
         <p className="hero__subtitle">
-          Rust, JavaScript, and parsers are all hard to learn,
-          let's combine these three and challenge ourselves to write a JavaScript parser in Rust.
+          <Translate id="homepage.subtitle.first">
+            Rust, JavaScript, and parsers are all hard to learn, let's combine
+            these three and challenge ourselves to write a JavaScript parser in
+            Rust.
+          </Translate>
         </p>
         <p className="hero__subtitle">
-          This will be the guide for you if you are interested in learning Rust, parsers,
-          or would like to contribute to <a href="https://github.com/boshen/oxc" target="__blank">oxc</a>,
-          <a href="https://swc.rs" target="__blank">swc</a> or <a href="https://rome.tools" target="__blank">Rome</a> in the near future.
```

Comment:
In order to translate the sentence here, it was necessary to stop using the a tag. Is this a problem...?

Reply:
This is ok.